cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -183,42 +238,83 @@ def is_vm_analysis_finished(vm_name):
     return True
 
 
+def detect_system_type(vm):
+
+    if getattr(vm, 'ssh'):
+        system_release = safe_string(vm.ssh.run_command("cat /etc/os-release").output)
+
+        all_systems_dict = RPM_BASED.values() + DEB_BASED.values()
+        for x in all_systems_dict:
+            if x['id'].lower() in system_release.lower():
+                return x
+    else:
```

Comment:
Does the WINDOWS system have SSH enabled? I am guessing so? Else we'd need a try,except here right?